x-pack/plugins/spaces/index.js

```diff
@@ -41,6 +41,9 @@ export const spaces = (kibana) => new kibana.Plugin({
     }],
     hacks: [],
     mappings,
+    savedObjectsSchema: {
+      namespaceAgnosticTypes: ['space']
```

Comment:
Are you expecting this schema to change in the future? Probably shift to look more like:

```js
savedObjectsSchema: {
  space: {
    isNamspaceAgnostic: true,
    mappings: ...,
  }
}
```

If so, maybe we can use that base structure now and implement the `[type].mappings` property down the road?

Reply:
I like this a lot better, thanks Spencer.